test/cypress/plugins/index.ts

```diff
@@ -0,0 +1,49 @@
+/// <reference type="cypress"
+const knex = require('knex')
+const DB = knex({
+  client: 'mysql',
+  connection: {
+    host: 'localhost',
+    user: 'root',
+    password: 'wordpress',
+    database: 'wordpress',
+  },
+  pool: { min: 1, max: 2 },
+})
+
+// ***********************************************************
+// This example plugins/index.js can be used to load plugins
+//
+// You can change the location of this file or turn off loading
+// the plugins file with the 'pluginsFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/plugins-guide
+// ***********************************************************
+
+// This function is called when a project is opened or re-opened (e.g. due to
+// the project's config changing)
+
+/**
+ * @type {Cypress.PluginConfig}
+ */
+// eslint-disable-next-line no-unused-vars
+module.exports = (on, config) => {
+  // `on` is used to hook into various events Cypress emits
+  // `config` is the resolved Cypress config
+  console.log('config', config)
```

Comment:
Is this console log meant to still be here?

Reply:
No, that's was just for me being nosey into the resolved config.